utility/build-homebrew.sh

```diff
@@ -44,21 +38,17 @@ class Cumulusci < Formula
 
   depends_on "python@3.8"
 
-$(cat "$RES_FILE")
-
   def install
     xy = Language::Python.major_minor_version "python3"
     site_packages = libexec/"lib/python#{xy}/site-packages"
     ENV.prepend_create_path "PYTHONPATH", site_packages
 
-    system "python3", *Language::Python.setup_install_args(libexec)
+    reqs = <<-REQS
+$(cat "$REQS_FILE")
+REQS
 
-    deps = resources.map(&:name).to_set
-    deps.each do |r|
-      resource(r).stage do
-        system "python3", *Language::Python.setup_install_args(libexec)
-      end
-    end
+    File.write("requirements.txt", reqs)
+    system "python3", "-m", "pip", "install", "-r", "requirements.txt", "--require-hashes", "--no-deps", "--prefix", libexec
```

Comment:
So [`setup_install_args`](https://github.com/Homebrew/brew/blob/b024954a0f203a4f439cc95eefb1180c96e943ce/Library/Homebrew/language/python.rb#L87) passes `--single-version-externally-managed` to `setup.py`. I don't understand what purpose it serves, so I'm unsure whether it's needed.

Reply:
`--single-version-externally-managed` tells setuptools to install the package in the form we're used to from pip (package folders plus a separate folder with the distribution metadata) instead of in an egg. The reason we're used to it is that pip started as a wrapper calling setuptools with this flag (see https://carljm.github.io/pipvirtualenv-preso/ for lots of details from ca. 2012), and I think it still does in certain legacy cases, but not when using wheels.